index.bs

```diff
@@ -643,6 +643,21 @@ partial interface Gamepad {
 <dfn attribute for="Gamepad" id="gamepad-getvrdisplays-attribute">displayId</dfn>
 Return the {{VRDisplay/displayId}} for the associated {{VRDisplay}}.
 
+# Processing Model # {#proc}
+
+## submitFrame ## {#submitframe-proc}
+
+The are are multiple situations where applications won't invoke submitFrame at the expected rate (the VRDisplay native framerate). The User Agent must intervene to protect the user from motion sickness or to consistently inform her about different situations like errors or navigation between pages. This is the breakdown of the different scenarios:
+
+* The application calls submitFrame at the expected rate but a few frames are dropped. The User Agent will re-project the last submitted frame.
+
+* The application calls submitFrame consistently below the expected framerate to a degree that could make the user feel umconfortable. The User agent will show a semi transparent overlay on top of the application with a notification icon (an hourglass for instance) until the application improves performance. If the low framerate persists for a long time the User Agent will show an opaque overlay indicating that the application is too slow to be presented.
+
+* An application can stop calling submitFrame all toghether under several circumstances:
+    * A content exception or error has ocurred. In this case the User Agent will notify the user with an opaque overlay.
+    * The User Agent transitions to a different site that will eventually call submitFrame (the User Agent will statically determine if a site will call submitFrame by the presence of meta name="vr"). The User Agent will render an opaque overlay with a loading spinner to inform the user of the situation. Once the new site starts calling submitFrame a semi transparent overlay with the new URL will be shown to notify the user of the site transition.
```

Comment:
The meta tag is uniquely associated with the interstitial, as we need such a mechanism to statically determine that a site will present WebVR content in response to the onvrdisplayactivate event.  In this case either an opaque intersistial or continual rendering of the prior site would be necessary to avoid the flash of 2d content during the transition.  Perhaps the concept of "flash of 2d content" included in the text may help give context?